mitodl/salt-ops
# Installing and configuring vector

#### What are the relevant tickets?
#1504

#### What's this PR do?
Install and configure vector on rabbit mq, xqwatcher, cas, reddit, and odlvideo.
Made global configurations for capturing host metrics, auth logs, and nginx logs.

#### How should this be manually tested?
Verify that vector is installed and running on managed nodes. 
Verify that expected configuration files land in /etc/vector and that their contents is accurate. 

#### What GIF best describes this PR or how it makes you feel?
![Fire](https://media.giphy.com/media/69hh4034OYSUo/giphy.gif)
